Screens/SimpleMenuScreen.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections.Generic;
+using Silksong.ModMenu.Elements;
+
+namespace Silksong.ModMenu.Screens;
+
+/// <summary>
+/// A basic menu screen with a single VerticalGroup for the content panel. Sufficient for most simple menus with not many elements.
```

Comment:
Perhaps the docs should opine on how many "not many" is? I believe the answer is probably 8 or 9 but I'm not sure

(Side note, out of scope for this PR but we may want to add a scrollable menu screen eventually)